### PR TITLE
Make repository usable as a template project

### LINKED ISSUE #1: Clean up repo and add clear instructions for use
Replace all references to `OverclockedSwerve` with `frc-java-junit-template`. Add instructions to README both (1) for teams without an existing project, who want to use this as a template for a new project, and also (2) for teams who have an existent project who want to integrate testing using this project as an example.

### PR DESCRIPTION
**Pull request by michaelsilver**
Note: uses the old version of WPILib

Close #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
-[![Build Status](https://travis-ci.org/246overclocked/OverclockedSwerve.svg?branch=master)](https://travis-ci.org/246overclocked/OverclockedSwerve)
-[![Stories in Ready](https://badge.waffle.io/246overclocked/OverclockedSwerve.svg?label=ready&title=Ready)](http://waffle.io/246overclocked/OverclockedSwerve)
+# FRC Java JUnit Template
+[![Build Status](https://travis-ci.org/246overclocked/frc-java-junit-template.svg?branch=master)](https://travis-ci.org/246overclocked/frc-java-junit-template)
 
-A clean version of our swerve drive code designed to be easily integrated into a new project.
+A template Java project for FRC using JUnit and Ant for building and unit testing.
+
+## Who should use this repository
+Whether you're an FRC team that wants to make a new project or build on an existing one, this the purpose of this repo is to provide an example as a template to integrate JUnit and Travis Continuous Integration into a Java FRC project. [TravisCI](https://travis-ci.org/) is a website that integrates with GitHub that will automatically build and test your code every time you push to GitHub.
```
Comment (PaulTerrasi) [inline README.md]: > this the purpose of this repo

Typo

File: README.md [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+
+## To use JUnit in your project
+  1. Clone the repo: `$ git clone https://github.com/246overclocked/frc-java-junit-template.git`
+  2. Make a new FRC Java project in Eclipse, or find your existing FRC project
+  3. Replace the your `build.xml` with the `build.xml` in this repository
+  4. Copy the `lib/` folder in this repository into the corresponding location in your FRC Java project
+  5. Open `lib/wpilib/wpilib.properties` and change 246 in the line `team-number=246` to your team's number
+  6. Create a `test` folder next to `src`, and create packages there with the same name as packages in `src`. Add test files that end in `Test.java` in the appropriate packages in the `test` folder. Eclipse can help you set up JUnit tests automatically. See [this guide](https://courses.cs.washington.edu/courses/cse143/11wi/eclipse-tutorial/junit.shtml) for more information.
+
```
Comment (PaulTerrasi) [inline README.md]: You might want to add comments on changing the names of the 2 launch files and the modified .gitignore file if using git.

**Comment by Owen-Gillespie** [comment]
Things to change:  Clarify when installing ant that the addition to path and new system variables should be systemwide, not the users.  Also add a link to instructions from screenstepslive for creating a FRC project and change the wording about where to put the lib folder to be clear that it goes in the top level project folder

File: README.md (hunk 2 above)
Comment (michaelsilver) [inline README.md]: Other than just renaming the files, how did you modify the two launch files? I just did a search in the two files for whatever the old project name was (OverclockedSwerve) and changed it to the new one (frc-java-junit-template). Is there a better way to do this?

**Comment by michaelsilver** [comment]
Thank you @Owen-Gillespie and @PaulTerrasi for your review. I think I have addressed all your concerns. How does it look to you?

**Comment by Owen-Gillespie** [comment]
LGTM!

**Comment by Owen-Gillespie** [comment]
LGTM!

**Pull request merged**